runtime/v2/shim/util.go

```diff
@@ -121,17 +123,8 @@ func WritePidFile(path string, pid int) error {
 	if err != nil {
 		return err
 	}
-	tempPath := filepath.Join(filepath.Dir(path), fmt.Sprintf(".%s", filepath.Base(path)))
-	f, err := os.OpenFile(tempPath, os.O_RDWR|os.O_CREATE|os.O_EXCL|os.O_SYNC, 0666)
-	if err != nil {
-		return err
-	}
-	_, err = fmt.Fprintf(f, "%d", pid)
-	f.Close()
-	if err != nil {
-		return err
-	}
-	return os.Rename(tempPath, path)
+
+	return continuity.AtomicWriteFile(path, []byte(strconv.Itoa(pid)), 0666)
```

Comment:
Sorry for the unclear comment in https://github.com/containerd/containerd/pull/8151#discussion_r1114401385

I don't want to change the original atomic write here, because we don't need random filename here and it won't be called twice for a container.

But we can use `WriteString(strconv.Itoa(pid))` here.

Reply:
got it, as soon as I can

Reply:
> @helen-frank: Cannot trigger testing until a trusted user reviews the PR and leaves an `/ok-to-test` message.

Just want to ask a question, what thresholds do you need to become a member of the containerd community? @fuweid 
( I may have been studying containerd recently, so I may make some contributions and hope to pass the online test faster.😀

Reply:
please checkout this one https://github.com/containerd/project/blob/main/GOVERNANCE.md#adding-maintainers~

Reply:
got it, It seems that I still have a long way to go.😀

Reply:
I need ok-to-test @fuweid